doc/format.rst

```diff
@@ -597,6 +597,24 @@ Other points to keep in mind
   (i.e. scalar types, sequence types), those arguments can be documented
   with type `array_like`.
 
+* Links : If you need to include hyperlinks in your docstring, note that
+  some docstring sections are not parsed as standard reST, and in these
+  sections, numpydoc may become confused by hyperlink targets such as::
+
+      .. _Example: http://www.example.com
+
+  If the Sphinx build issues a warning of the form
+  ``WARNING: Unknown target name: "example"``, then that is what is happening.
+  There are two possible workarounds.  First, the inline form can be used::
+
+      `Example <http://www.example.com>`_
+
+  Second, the hyperlink target can be placed in a section that *is* parsed
```

Comment:
I suggest we only recommend a version that will always work; if we choose to parse the Notes section differently later on, this may break.  Others may disagree; I don't feel strongly.

Reply:
I'm OK with that.